core/kysely/database-init.ts

```diff
@@ -0,0 +1,86 @@
+import type { LogEvent } from "kysely";
+
+import { Kysely, PostgresDialect } from "kysely";
+import pg from "pg";
+
+import type { Database } from "db/Database";
+import { databaseTables } from "db/table-names";
+import { logger } from "logger";
+
+import { ArtificialLatencyPlugin } from "./artificial-latency-plugin";
+import { UpdatedAtPlugin } from "./updated-at-plugin";
+
+type DatabaseOptions = {
+	url: string;
+	debug?: boolean;
+	logLevel?: string;
+	latency?: number;
+	env?: string;
```

Comment:
```suggestion
	nodeEnv?: string;
```

`env` means too many different things already imo